rollup.config.js

```diff
@@ -16,6 +16,16 @@ const name = 'Rifm';
 const getBabelOptions = ({ useESModules }) => ({
   exclude: '**/node_modules/**',
   runtimeHelpers: true,
+  presets: [
+    [
+      '@babel/env',
+      {
```

Comment:
Write in one line please

Reply:
I can't - its on prettier how to format js

Reply:
Try again. Prettier allows to write objects in one line.